tests/mock-snapd.h

```diff
@@ -315,6 +316,9 @@ void            mock_snap_set_disabled            (MockSnap      *snap,
 
 gboolean        mock_snap_get_disabled            (MockSnap      *snap);
 
+void            mock_snap_set_donation            (MockSnap      *snap,
```

Comment:
No tests seem to use this - you'll need to update the existing tests that use these fields.